orchagent/intfsorch.cpp

```diff
@@ -731,6 +731,7 @@ void IntfsOrch::doTask(Consumer &consumer)
                     {
                         IntfsEntry intfs_entry;
                         intfs_entry.ref_count = 0;
+                        intfs_entry.proxy_arp = false;
```

Comment:
Is this required? For `lo `interfaces, we are not continuing with the set flow. So just curious why initialize here?

Reply:
Just for object data member init completeness. I think it does no harm.